sci-libs/symmetrica/symmetrica-2.0.ebuild

```diff
@@ -31,12 +21,10 @@ src_prepare() {
 #	plus documentation that you can use as you wish in your programs.
 #	For sage and ease of use we make it into a library with the following
 #	makefile (developped by F. Bissey and T. Abbott (sage on debian).
-	cp "${FILESDIR}/makefile" "${S}/makefile"
+	cp "${FILESDIR}/makefile" "${S}/makefile" || die
 }
 
 src_install() {
-	emake DESTDIR="${D}" install || die "make install failed"
-
-#	symmetrica's documentation are just text files with a ".doc" extension.
-	dodoc README *.doc
+	default
+	use doc && dodoc *.doc
```

Comment:
It's done same in the [sage overlay](https://github.com/cschwan/sage-on-gentoo/blob/master/sci-libs/symmetrica/symmetrica-2.0-r4.ebuild), so I think we should leave it as is.